app/views/planning_applications/site_notices/_status.html.erb

```diff
@@ -17,7 +17,7 @@
 <div class="govuk-!-display-none">
   <span id="application-reference"><%= @planning_application.reference %></span>
   <div id="site-notice-content">
-    <div style="width: 430px;">
+    <div style="width: 430px; font-family: Arial;">
```

Comment:
Putting Arial in the `govuk-font-family` instead of here might also work, as a fallback after open sans, but it occurred to me too late and it's probably not worth messing further with it for the time being.